platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/MachineService.java

```diff
@@ -511,7 +512,8 @@ private MachineDescriptor toDescriptor(Instance machine) throws ServerException
                                             .withRef(serverEntry.getValue().getRef())
                                             .withUrl(serverEntry.getValue().getUrl()));
         }
-        machineDescriptor.withMetadata(machine.getMetadata().getProperties())
+        machineDescriptor.withProperties(machine.getMetadata().getProperties())
+                         .withMetadata(newDto(MachineMetadataDto.class).withEnvVariables(machine.getMetadata().getEnvVariables()))
```

Comment:
I think that <code>.withEnvVariables(machine.getMetadata().getEnvVariables()))</code> should be in a new line